Makefile

```diff
@@ -0,0 +1,2 @@
+control.dud: control/control.xml
+	mkdud -c control.dud -d tw control
```

Comment:
well, if we create such file, we should probably also have it in `.gitignore`. But it is not critical.